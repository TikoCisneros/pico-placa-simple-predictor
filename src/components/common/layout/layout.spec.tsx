import React from 'react';
import { render } from '@testing-library/react';
import Layout from './layout';

describe('Layout test', () => {
  it('should renders correctly', () => {
    const { asFragment } = render(<Layout />);
    expect(asFragment()).toMatchSnapshot();
  });
});