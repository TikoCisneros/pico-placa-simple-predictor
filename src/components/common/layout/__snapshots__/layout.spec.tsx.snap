// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout test should renders correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root makeStyles-root-1 MuiGrid-container"
  />
</DocumentFragment>
`;
